projects/fsastorefrontlib/src/assets/translations/en/b2b.en.ts

```diff
@@ -5,3 +5,178 @@ export const b2b = {
     notAssigned: 'Not Assigned',
   },
 };
+
```

Comment:
Since Spartacus hadn't exported translations for B2B feature, we need to copy them to our lib. Hopefully this is temporary, and should be removed in future (when Spartacus exports translations).